doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -631,6 +633,15 @@ Modem libraries
 
   * Fixed a bug where AT responses would erroneously be written to the logging UART instead of being written to the chosen ``ncs,at-host-uart`` UART device when the :kconfig:option:`CONFIG_LOG_BACKEND_UART` Kconfig option was set.
 
+* :ref:`modem_info_readme` library:
+
+  * Added:
+
+    * The :c:type:`modem_info_data_type` type for representing LTE link information data types.
```

Comment:
```suggestion
    * The :c:enum:`modem_info_data_type` type for representing LTE link information data types.
```
with type the API don't link. I think enum would work.

Reply:
Thanks Divya, updated